iac-extensions/docker/src/main/java/org/sonar/iac/docker/utils/ArgumentUtils.java

```diff
@@ -76,6 +79,15 @@ private static String resolveExpression(Expression expression) {
     return null;
   }
 
+  @Nullable
+  public static SyntaxToken argumentToSyntaxToken(Argument argument) {
+    String value = ArgumentUtils.resolve(argument).value();
+    if (value != null) {
+      return new SyntaxTokenImpl(value, argument.textRange(), Collections.emptyList());
+    }
+    return null;
+  }
```

Comment:
Please add a comment. We should remove this helper after we drop all `literals()` methods.